koopovereenkomst-v3-simple/pages/steps_verkoper/Step5.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import Typography from "@mui/material/Typography";
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import { Box } from "@mui/system";
+
+export default function Step5({ step = 5, handleNext, handleBack = () => {} }) {
+  return (
+    <Box sx={{ flex: 1 }}>
+      <Typography variant="h1" color="text.primary" align="center">
+        Start een nieuwe koopovereenkomst
+      </Typography>
+      <Typography variant="h2" color="text.primary" align="center">
+        {step}. Aanmaken koopovereenkomst
+      </Typography>
+
+      <Stack direction="row" justifyContent="end">
+        <Button variant="contained" onClick={handleBack}>
+          Terug
+        </Button>
+        <Button variant="contained" onClick={handleNext}>
```

Comment:
D'r moet nog een keer een einde aan 'gebreien' worden, niet? 😁

Reply:
Heb ik het laatste setje commits een laatste step toegevoegd (6). Die heeft alleen een terugknop. Maar misschien willen we nog een aparte conclusie / contact / further research / terug naar main knop